runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -407,4 +412,28 @@ private static BufferedImage offset(final BufferedImage image, final float[] sca
 	{
 		return new RescaleOp(scales, offsets, null).filter(image, null);
 	}
+
+
+	/**
+	 * Converts the buffered image into a sprite image and returns it
+	 * @param image  The image to be converted
+	 * @param client Current client instance
+	 * @return       The buffered image as a sprite image
+	 */
+	public static SpritePixels getImageSpritePixels(BufferedImage image, Client client)
```

Comment:
this is also used in bank tag tabs, so you can use this there too

Reply:
`client.getSpriteOverrides().putAll(TabSprites.toMap(client));`
I already updated `TabSprites` so this should be good :)